### ISSUE TITLE
Reflectometry Server: Inst archive errors

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#6000]
As a support person I do not want the reflectometry server to fill up disk space with logs. Currently POLREF has a number of very large (~5 GB) ARINST log files filled with:

```
[2020-12-20 09:42:25] 2020-12-20 09:42:25.793 WARNING [Thread 32] org.csstudio.vtype.pv.jca.JCAContext (contextException) - Channel Access Exception from com.cosylab.epics.caj.CAJContext@45959977 for channel CAJChannel = { name = IN:POLREF:REFL_01:PARAM:S3S:SP.VAL, connectionState = DISCONNECTED }: Virtual circuit disconnect
[2020-12-20 09:42:25] 2020-12-20 09:42:25.794 WARNING [Thread 32] org.csstudio.vtype.pv.jca.JCAContext (contextException) - Channel Access Exception from com.cosylab.epics.caj.CAJContext@45959977 for channel CAJChannel = { name = IN:POLREF:REFL_01:PARAM:BENCHOFFSE:SP.VAL, connectionState = DISCONNECTED }: Virtual circuit disconnect
[2020-12-20 09:42:25] 2020-12-20 09:42:25.794 WARNING [Thread 32] org.csstudio.vtype.pv.jca.JCAContext (contextException) - Channel Access Exception from com.cosylab.epics.caj.CAJContext@45959977 for channel CAJChannel = { name = IN:POLREF:REFL_01:PARAM:A2CHI:SP.VAL, connectionState = DISCONNECTED }: Virtual circuit disconnect
[2020-12-20 09:42:25] 2020-12-20 09:42:25.794 WARNING [Thread 32] org.csstudio.vtype.pv.jca.JCAContext (contextException) - Channel Access Exception from com.cosylab.epics.caj.CAJContext@45959977 for channel CAJChannel = { name = IN:POLREF:REFL_01:PARAM:SMANGLE.VAL, connectionState = DISCONNECTED }: Virtual circuit disconnect
[2020-12-20 09:42:25] 2020-12-20 09:42:25.794 WARNING [Thread 32] org.csstudio.vtype.pv.jca.JCAContext (contextException) - Channel Access Exception from com.cosylab.epics.caj.CAJContext@45959977 for channel CAJChannel = { name = IN:POLREF:REFL_01:PARAM:FOFFSET:SP.VAL, connectionState = DISCONNECTED }: Virtual circuit disconnect
```

## Acceptance Criteria
* The above messages no longer appear in the inst archiver log

**Comment by John-Holt-Tessella** [comment]
If this is not easy to fix we should be able to mitigate it by moving the file (check it is not being written to the inst archive database)

**Comment by FreddieAkeroyd** [comment]
Had a discussion with @ThomasLohnert - there seem to be `max_array_bytes` error messages mentioned at some points that looks to be potentially related. It seems a bit extreme, but it looks like CSS may disconnect all PVs on an IOC when it receives a bad request. There is a char array PV on the refl server, but it is only 10k is size and the error mentions arrays over 64k. Maybe it is a corrupt data packet, but potentially an error in `pcaspy` occasionally putting the wrong value in the array packet? Conclusion of our discussion was that the char array PV is actually log messages that are going to the IOC log anyway, so they don't need to be archived in ARINST. So we'll remove the archive option from this PV - that should mean there are no array pvs being archived and so hopefully then is then no chance of getting a spurious array size exceeded error.

**Comment by ThomasLohnert** [comment]
https://github.com/ISISComputingGroup/EPICS-refl/pull/25
https://github.com/ISISComputingGroup/IBEX/pull/6111

**Comment by FreddieAkeroyd** [comment]
For info a typical array error message was
```
[2020-12-21 00:00:00] 2020-12-21 00:00:00.669 SEVERE [Thread 33] com.cosylab.epics.caj.impl.CATransport (processRead) 
 - Received payload size (80816) is larger than configured maximum array size (65536), disconnecting...
```
though the payload size mentioned varies.

**Comment by ThomasLohnert** [comment]
In case this does not get resolved, a summary of what I found so far:
- The errors happened 4 or 5 times during cycle and seemed to occur at random times. The instarchiver log would then get flooded with these `virtual circuit disconnect` messages for each parameter PV for a duration between 1-10 minutes at a time, then it would stop again.
- The `virtual circuit disconnect` errors are always accompanied by several messages like the one in Freddie's comment above. Not sure if that is causation or just correlation. One thing I noticed is that the "Received Payload Size" of the message exceeding the max array size would grow over time.
- I was unable to reproduce the error on my local machine. Played around with the options in `CSS/master/ArchiveEngine/mysql_settings_inst.ini`:
    - `org.csstudio.diirt.util.core.preferences/diirt.ca.monitor.mask` - tried setting this to different options
    - `org.csstudio.diirt.util.core.preferences/diirt.ca.max.array.size` - decreased this to trigger the max array bytes error, but I did not see the virtual circuit disconnect 
    - `org.csstudio.diirt.util.core.preferences/diirt.ca.beacon.period`- tried making this shorter
- Also tried just filling up the `REFL_01:LOG` PV to fill it up to see if it could somehow get above the max array size, but it starts throwing a different error once you hit the 10k character limit
- I think it's possible the errors get thrown after some external event killing the channel access layer, but not sure why this only affects reflectometry server PVs. Tom suggested the errors could be  to pcaspy PVs missing some fields that normal PVs have. So far I have not found anything conclusive (also the error gets specifically thrown on the `.VAL` channels which all our PVs definitely should have)
